<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,8">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,3,1,0,14">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="labe_1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="images.qrc">:/new/—Pngtree—cartoon check symbol free illustration_4599313.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="S_checkBox" name="checkBox1">
        <property name="cursor">
         <cursorShape>ForbiddenCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton::indicator:checked

{
	
	
	background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:1, fx:0.5, fy:0.5, stop:0 rgba(0,200,0,255), stop:0.479904 rgba(0,200,0,255), stop:0.522685 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
	

	
	
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="images.qrc">:/new/—Pngtree—flags and pennants_785825.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="F_checkBox" name="checkBox2">
        <property name="cursor">
         <cursorShape>ForbiddenCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton::indicator:checked
{
	
	
	background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:1, fx:0.5, fy:0.5, stop:0 rgba(0,200,0,255), stop:0.479904 rgba(0,200,0,255), stop:0.522685 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
	

	
	
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>9</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="currentText">
         <string>A*</string>
        </property>
        <item>
         <property name="text">
          <string>A*</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dijkstra</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BFS</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="pushButton" name="pushButton1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>ForbiddenCursor</cursorShape>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>Right-click the map to set &quot;Obstacles&quot;.
Left-click to set the &quot;Starting/Finishing Point&quot;.
</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="graphicsView" name="graphicsView1">
      <property name="toolTip">
       <string comment="&quot;Note: To reset, click the icon again.&quot;" extracomment="&quot;Note: To reset, click the icon again.&quot;">&quot;Note: To reset, click the icon again.&quot;</string>
      </property>
      <property name="toolTipDuration">
       <number>5</number>
      </property>
      <property name="statusTip">
       <string comment="&quot;Note: To reset, click the icon again.&quot;" extracomment="&quot;Note: To reset, click the icon again.&quot;">&quot; - Note : To reset, click the icon again.&quot;</string>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>graphicsView</class>
   <extends>QGraphicsView</extends>
   <header>graphicsview.h</header>
   <slots>
    <signal>flagChanged(bool,bool)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>S_checkBox</class>
   <extends>QRadioButton</extends>
   <header>s_checkbox.h</header>
  </customwidget>
  <customwidget>
   <class>F_checkBox</class>
   <extends>QRadioButton</extends>
   <header>f_checkbox.h</header>
  </customwidget>
  <customwidget>
   <class>pushButton</class>
   <extends>QPushButton</extends>
   <header>pushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
